pkgs/development/libraries/science/astronomy/indilib/indi-3rdparty.nix

```diff
@@ -38,18 +39,17 @@ stdenv.mkDerivation rec {
   ];
 
   postPatch = ''
-    for f in indi-qsi/CMakeLists.txt \
-             indi-dsi/CMakeLists.txt \
-             indi-armadillo-platypus/CMakeLists.txt \
-             indi-orion-ssg3/CMakeLists.txt
+    for f in */CMakeLists.txt \
+             */*.rules;
```

Comment:
The udev rules don't need to be patched here, since we don't use them from this package.

Reply:
It seems they do; they are used from both packages apparently.